src/services/shims.ts

```diff
@@ -883,7 +883,7 @@ module ts {
                     return {
                         options: configFile.options,
                         files: configFile.fileNames,
-                        errors: realizeDiagnostics(configFile.errors, '\r\n')
+                        errors: [realizeDiagnostics(configFile.errors, '\r\n')]

```

Comment:
forwardJSONCall takes a function that returns any. so it is not going to complain regardless what we put in there :)